com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/RefractionUIBlock.cs

```diff
@@ -79,7 +79,7 @@ public override void LoadMaterialProperties()
         /// </summary>
         public override void OnGUI()
         {
-            if (refractionModel != null)
+            using (new EditorGUI.DisabledScope(refractionModel == null))
```

Comment:
i assume this changes the UI, is it on purpose?

Reply:
Yeah, before we wer not showing anything, now we show everything disabled, I think it makes more sense as we end up with a empty foldout.